src/openrct2/object/Object.h

```diff
@@ -29,6 +29,9 @@ constexpr const ObjectEntryIndex OBJECT_ENTRY_INDEX_NULL = std::numeric_limits<O
 struct ObjectRepositoryItem;
 using ride_type_t = uint16_t;
 
+constexpr const int8_t VersionNumFields = 3;
+using ObjectVersion = std::tuple<uint16_t, uint16_t, uint16_t>;
```

Comment:
The hardcoded 3 can be removed, or at least verified to match the actual type, by using `std::tuple_size`.

```suggestion
constexpr const int8_t VersionNumFields = 3;
using ObjectVersion = std::tuple<uint16_t, uint16_t, uint16_t>;
static_assert(std::tuple_size<ObjectVersion>{} == VersionNumFields );
```